arch/x86_64/symbol.c

```diff
@@ -93,6 +93,7 @@ int arch_load_dynsymtab_noplt(struct uftrace_symtab *dsymtab, struct uftrace_elf
 		pr_dbg3("[%zd] %c %lx + %-5u %s\n", dsymtab->nr_sym, sym->type, sym->addr,
 			sym->size, sym->name);
 	}
+	sort_dynsymtab(dsymtab);
```

Comment:
I think this change can be applied separately.

Reply:
Oh, actually I meant the `sort_dynsymtab()` only.  I think you can split the changes in the following way.

1. add `sort_dynsymtab()` at the end of `arch_load_dynsymtab_noplt()`.  It's an independent issue and can go separately.
2. add `arch_load_dynsymtab_noplt()` to `load_dynsymtab()`.  This should have no effect to normal PLT binaries.
3. check PLT with additional skip symbols and read from `.rela.dyn`.